products/mageia.md

```diff
@@ -0,0 +1,39 @@
+---
+title: mageia
+category: os
+permalink: /mageia
+releasePolicyLink: https://www.mageia.org/en/support/
+releaseDateColumn: true
+eolColumn: End of Life
+versionCommand: cat /usr/lib/os-release
+changelogTemplate: https://wiki.mageia.org/en/Mageia___RELEASE_CYCLE___Release_Notes
+releaseLabel: "mageia __RELEASE_CYCLE__"
+
+releases:
+-   releaseCycle: "8"
+    releaseDate: 2021-02-26
+-   releaseCycle: "7"
+    eol: 2021-06-30
+    releaseDate: 2019-07-01
+-   releaseCycle: "6"
+    eol: 2019-09-30
+    releaseDate: 2017-07-16
+-   releaseCycle: "5"
+    eol: 2017-12-31
+    releaseDate: 2015-06-19
+-   releaseCycle: "4"
+    eol: 2015-09-19
+    releaseDate: 2014-02-01
+-   releaseCycle: "3"
+    eol: 2014-11-26
+    releaseDate: 2013-05-19
+-   releaseCycle: "2"
+    eol: 2013-11-22
+    releaseDate: 2012-05-22
+-   releaseCycle: "1"
+    eol: 2012-12-01
+    releaseDate: 2011-06-01
+
+---
+
+> [mageia](https://www.mageia.org/en/) is a GNU/Linux-based, Free Software operating system. It is a community project, supported by a nonprofit organisation of elected contributors.
```

Comment:
Capital

```suggestion
> [Mageia](https://www.mageia.org/en/) is a GNU/Linux-based, Free Software operating system. It is a community project, supported by a nonprofit organisation of elected contributors.
```

Reply:
@captn3m0 why are are still keeping /en/ part in here ?